.circleci/config.yml

```diff
@@ -767,7 +767,7 @@ jobs:
 
   build_docs:
     # deploy official documentation
-    executor: python38
+    executor: ddtrace_dev
```

Comment:
We expect #1711 to have updated the `datadog/dd-trace-py` image as it will include the `libenchant-dev` necessary for `sphinxcontrib-spelling`.